pkg/querier/worker/frontend_processor.go

```diff
@@ -59,19 +65,24 @@ func (fp *frontendProcessor) notifyShutdown(ctx context.Context, conn *grpc.Clie
 }
 
 // runOne loops, trying to establish a stream to the frontend to begin request processing.
```

Comment:
that comment seems outdated, it mentions `runOne`

Reply:
Fixed, thanks!